ajax/libs/floating.js/package.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "floating.js",
+  "description": "Float a number of things up on a page (hearts, flowers, ...)",
+  "homepage": "https://haroen.me/floating.js/",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/Haroenv/floating.js.git"
+  },
+  "author": "Haroen Viaene",
+  "license": "Apache-2.0",
+  "filename": "floating.js",
+  "keywords": [
```

Comment:
Please modify the keywords to let users be able to find the library in a easier way. You can test if the keywords are good by googling them. If GitHub or homepage displays on first page of search results, then it's good. For instance, `javascript`, `element`, `float`, `infinitely`.